src/main/client/src/dashBoard/bookList/BookList.js

```diff
@@ -17,9 +17,11 @@ class BookList extends Component {
               updateBook={this.props.updateBook}
               editStateChange={this.props.editStateChange}
               reserveBook={this.props.reserveBook}
+              checkBook={this.props.checkBook}
               key={book.id}
               book={book}
               reservePopText={this.props.reservePopText}
+              checkBookPopText={this.props.availability}
```

Comment:
These could probably be combined into one variable since they are the same in the state